guides/team-and-project-scheduling.md

```diff
@@ -91,11 +91,9 @@ The **project lead** should:
 ### Reporting our time
 
 Every team member working on a client or named project is responsible for
-reporting their time, as mentioned
-[in the Playbook](http://playbook.dxw.com/#/?id=reporting-our-time). More on how
-to do this in the
-[reporting time](http://playbook.dxw.com/#/guides/team-and-project-scheduling?id=reporting-time)
-section of this guide.
+reporting their time, as mentioned [in the Playbook](/#reporting-your-time).
+More on how to do this in the [reporting time](#reporting-time) section of this
```

Comment:
Note to self to check if reporting your time is still a separate guide as well as an internal anchor link.